Project/Headers/main.h

```diff
@@ -1,13 +0,0 @@
-/* Define to prevent recursive inclusion -------------------------------------*/
```

Comment:
Jesi ovo namjerno izbrisao ili? Mislim da nebi bilo loše da svo uključivanje vanjskih library-a bude kroz main.h tako da imamo sve na jednom mjestu, pa da onda u pojedinačne source fileove uključujemo samo main.h?

Reply:
Ma ne, main.c i main.h su greške, tek sad to vidim